src/com/xilinx/rapidwright/design/ModuleInst.java

```diff
@@ -389,6 +390,12 @@ public boolean place(Site newAnchorSite, boolean skipIncompatible, boolean allow
                 if (pipSet != null) {
                     pipSet.add(newPip);
                 }
+                // Some tiles have nodes that are depopulated, we need to detect those
+                Node endNode = newPip.getEndNode();
+                if (endNode != null && endNode.getAllDownhillPIPs().size() == 0
```

Comment:
Do you know if `Node`s can be partially depopulated, or that they must either be fully depopulated or fully populated with no middle ground?

Reply:
The problem here arises because the module uses a node in the original placed context that spans a tile type that is different in the problematic placed context.  The tile type simply doesn't have the connection as it did in the original context.  That means the tile/wire participates in a different node.

Reply:
Sorry, let me reprhase my question: Can a node simply have one arc depopulated, or must it have all arcs depopulated (which is what the above code checks for)? The former would be painful to identify.

Reply:
I have not performed an exhaustive architectural search so I don't know.  There are actually many differences when comparing one `INT` tiles node fanout to another.  The reason this one became an issue was because the fanout involved a tile that had no other presence from the module except the fact that this node passed through it.  Most of the time, the module's footprint is pervasive in that it covers all the surrounding tiles so that if there is a change in the fanout, the tile type will have already been compared and the candidate location would be disqualified.